src/backend/test/unit/common/helpers/date-helper.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { shortDate } from 'src/common/helpers/date-helper';
+
+/**
+ * @Description
+ * This file tests the contents of common/helpers/data-helper.ts
+ */
+describe('DateHelper', () => {
+  /**
+   * @method shortDate
+   *
+   * @description
+   * This test suite validates that the method returns expected formatted data
```

Comment:
Would it be better to elaborate what the test is testing in terms of input/output

input: give a valid Date object 
          e.g. new Date('Wed Dec ....')
output: the date returned must be of the formant mm/dd/yy

What if we give a different input- null, random string, plain numbers, or a date in the short form but different formatting dd/mm/yyyy. What outputs should we expect?

If a new developer needs to expand this test case can we add to the description possible test cases to cover? 

Food for thought.

Reply:
> Would it be better to elaborate what the test is testing in terms of input/output

Cool, sure we can. I have added the input and outputs to the comments. <br/><br/>

> What if we give a different input- null, random string, plain numbers, or a date in the short form but different formatting dd/mm/yyyy. What outputs should we expect?

Updated the comment to reflect the null case to include defaults. random string, plain numbers, or a date in the short form won't be an accepted type. Typescript will take care of that for new devs.

And regarding the default date, we really can't test it as the current date is dynamic when the test runs. We may have to write similar logic to convert it to the date string, and the code to test code does not make sense and is an anti-pattern